polaris.shopify.com/content/components/actions/button.md

```diff
@@ -27,21 +27,18 @@ examples:
   - fileName: button-plain.tsx
     title: Plain
     description: Use for less important or less commonly used actions since they’re less prominent. For example, plain buttons are used as actions in cards.
-  - fileName: button-plain-monochrome.tsx
-    title: Plain monochrome
-    description: Use for less important or less commonly used actions where matching the current text color is desired. For example in the InlineError component.
-  - fileName: button-plain-destructive.tsx
-    title: Plain destructive
-    description: Use for actions that will delete merchant data or be otherwise difficult to recover from. Since they’re less prominent, use for less important or less commonly used destructive actions. For example, plain buttons are used as actions in cards.
+  - fileName: button-tertiary.tsx
+    title: Tertiary
+    description: Use for less important or less commonly used actions where matching the default text color is desired.
```

Comment:
@sarahill is there a better description for the new `tertiary` variant we can add here?

Reply:
How about something like:

Use for tertiary actions and can be paired with any other button style to create visual variation.